auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -255,6 +272,19 @@ public Builder withScheme(@NonNull String scheme) {
             return this;
         }
 
+        /**
+         * Specify a custom Redirect Uri to use to invoke the app on redirection.
```

Comment:
Same as above. There are 2 instances in this doc comment.

Reply:
```suggestion
         * Specify a custom Redirect URI to use to invoke the app on redirection.
```